docs: Improve Run a node - Overview (#112)
oasisprotocol/docs: docs/general/run-a-node/node-operator-overview.mdx

# Overview

The following documentation is intended to assist any individual or 

> organisation in participating in the Oasis network as a node operator. To join 

Reviewer: We use US English.

```suggestion
organization in participating in the Oasis network as a node operator. To join 
```

> the network we recommend you first try running nodes on the Testnet. The 

Reviewer: ```suggestion
the network we recommend you first try running a node on the Testnet. The 
```

> Testnet is a playground, where you can learn and experiment without the risk 

Reviewer: ```suggestion
Testnet is a playground where you can learn and experiment without the risk 
```

> of loosing real tokens. 

Reviewer: ```suggestion
of losing real tokens. 
```

operators can be seen on a block explorer such as [Oasis Scan].

:::

:::info

Operating a ParaTime Node on the Mainnet requires the participation of node 
operators who have the validator node in the active validator set. ParaTimes 
have their own reward system, participation requirements and structure. As a 
node operator you can participate in any number of ParaTimes.

:::

If you have any questions about running a node you can find us on [Discord].

[Oasis Network]: ../oasis-network/overview.md
[Oasis Scan]: https://www.oasisscan.com/validators
[Discord]: https://discord.gg/RwNTK8t

## Set Up Your Node overview

To run a **Validator node**, make sure your system meets the [Hardware] and the 
[System] prerequisites and has [Oasis Node] installed.

Then proceed by following the [Run a Validator Node] guide to:

* Create your entity.
* Initialize and configure your node.
* Put enough stake in your escrow account.
* Register your entity on the network.

To run a **ParaTime node** make sure to first set up a Validator node. Then set 
up a [Trusted Execution Environment (TEE)]. After that proceed to the [Run a 
ParaTime node].

> 

Reviewer: Here, I would mention the third option - client nodes. Add links where appropriate.

```suggestion

If you are building a service on top of the Oasis network, we suggest
that you set up a **non-validator node** and (optionally) a **ParaTime client
node**, so your service do not depend on third party endpoints which are
often behind a traffic limiter.
```

governance - the network started with 80 nodes in the validator set in 2020 and 
has expanded to 120 nodes over the past few network upgrades. Current node 
operators can be seen on a block explorer such as [Oasis Scan].

:::

:::info

Operating a ParaTime Node on the Mainnet requires the participation of node 
operators who have the validator node in the active validator set. ParaTimes 
have their own reward system, participation requirements and structure. As a 
node operator you can participate in any number of ParaTimes.

:::

If you have any questions about running a node you can find us on [Discord].

[Oasis Network]: ../oasis-network/overview.md
[Oasis Scan]: https://www.oasisscan.com/validators
[Discord]: https://discord.gg/RwNTK8t

## Set Up Your Node overview

To run a **Validator node**, make sure your system meets the [Hardware] and the 
[System] prerequisites and has [Oasis Node] installed.

Then proceed by following the [Run a Validator Node] guide to:

* Create your entity.
* Initialize and configure your node.
* Put enough stake in your escrow account.
* Register your entity on the network.

> To run a **ParaTime node** make sure to first set up a Validator node. Then set 

Reviewer: ```suggestion
To run a **ParaTime node** make sure to first set up a Validator node. Then, set 
```

> up a [Trusted Execution Environment (TEE)]. After that proceed to the [Run a 

Reviewer: ```suggestion
up a [Trusted Execution Environment (TEE)], if you want to run confidential ParaTimes. After that, proceed to the [Run a 
```

ParaTime node].

> If you are building a service on top of the Oasis Network, we suggest

Reviewer: ```suggestion
Finally, if you are building a service on top of the Oasis Network, we suggest
```

> that you set up a [non-validator node] and (optionally) a [ParaTime client

Reviewer: ```suggestion
that you set up your own **[Non-validator node]** and (optionally) a **[ParaTime client
```

> node], so your service do not depend on third party endpoints which are

Reviewer: ```suggestion
node]**, so that your service do not depend on any third party endpoints which are
```

often behind a traffic limiter.

[Hardware]: ./prerequisites/hardware-recommendations.md
[System]: ./prerequisites/system-configuration.mdx
[Oasis Node]: ./prerequisites/oasis-node.md
[Run a Validator Node]: ./set-up-your-node/run-validator.md

> [Trusted Execution Environment (TEE)]: ./prerequisites/set-up-trusted-execution-environment-tee.md

Reviewer: ```suggestion
[Trusted Execution Environment (TEE)]:
  ./prerequisites/set-up-trusted-execution-environment-tee.md
```

Author: If I apply this change link does not work, so I kept the old version.

[Run a ParaTime node]: ./set-up-your-node/run-a-paratime-node.mdx
[ParaTime client node]: ./set-up-your-node/run-a-paratime-node.mdx

> [non-validator node]: ./set-up-your-node/run-non-validator.md

Reviewer: ```suggestion
[Non-validator node]: ./set-up-your-node/run-non-validator.md
```
